src/NUnitEngine/nunit.engine.core/Drivers/DriverService.cs

```diff
@@ -70,9 +70,9 @@ public IFrameworkDriver GetDriver(AppDomain domain, string assemblyPath, string?
 
                 if (platform == "Silverlight" || platform == ".NETPortable" || platform == ".NETStandard" || platform == ".NETCompactFramework")
                     if (skipNonTestAssemblies)
-                        return new SkippedAssemblyFrameworkDriver(assemblyPath);
+                        return new SkippedAssemblyFrameworkDriver(assemblyPath, "X");
```

Comment:
Suggest to use `package.ID` here as well as otherwise the `ID` will not be unique.

Reply:
An experiment I ran before creating the final form of the driver. :-(